lib/iris/tests/test_plot.py

```diff
@@ -339,16 +334,19 @@ def test_circular_changes(self):
         self.check_graphic()
 
 
-@tests.skip_data
-@tests.skip_plot
-class Test1dFillBetween(tests.GraphicsTest):
-    def setUp(self):
-        super().setUp()
+@_shared_utils.skip_data
+@_shared_utils.skip_plot
+class Test1dFillBetween(_shared_utils.GraphicsTest):
+    @pytest.fixture(autouse=True)
+    def _set_draw_method(self):
```

Comment:
This class is inherited. The only difference in the inheriting class is `self.draw_method`. I was therefore able to separate this into its own fixture so that the rest of `_setup()` didn't need to be re-written in the inheriting class.

https://github.com/SciTools/iris/blob/098466411b26e8c539028569b55686fea1ad607c/lib/iris/tests/test_plot.py#L428-L433

BTW this is what I meant when I said I had seen the benefits of using fixtures rather than `setup_method()`.

Reply:
Awesome, happy with that.